tests/integration/common.rs

```diff
@@ -1,13 +1,158 @@
+use std::path::PathBuf;
+use std::sync::Arc;
+
+use derive_builder::Builder;
+use derive_more::{Deref, DerefMut};
+use sc_network_common::config::MultiaddrWithPeerId;
+use subspace_sdk::farmer::{CacheDescription, PlotDescription};
+use subspace_sdk::node::{chain_spec, ChainSpec, DsnBuilder, NetworkBuilder, Role};
+use tempfile::TempDir;
+
 pub fn setup() {
     let _ = tracing_subscriber::fmt()
         .with_env_filter(
             "debug,parity-db=info,cranelift_codegen=info,wasmtime_cranelift=info,\
              subspace_sdk=trace,subspace_farmer=trace,subspace_service=trace,\
              subspace_farmer::utils::parity_db_store=debug,trie-cache=info,wasm_overrides=info,\
-             libp2p_gossipsub::behaviour=info,wasmtime_jit=info,wasm-runtime=info"
+             jsonrpsee_core=info,libp2p_gossipsub::behaviour=info,wasmtime_jit=info,\
+             wasm-runtime=info"
                 .parse::<tracing_subscriber::EnvFilter>()
                 .expect("Env filter directives are correct"),
         )
         .with_test_writer()
         .try_init();
 }
+
+#[derive(Builder)]
+#[builder(pattern = "immutable", build_fn(private, name = "_build"), name = "NodeBuilder")]
+pub struct InnerNode {
+    #[builder(default)]
+    not_force_synced: bool,
```

Comment:
what is the reason for not using `force_synced` directly, but using `not_force_synced`?
in case of the value is false, double-negatives are making humans more prone to make mistakes

Reply:
Because, by default dev node is force synced, so I decided to allow override for tests, but with force synced set to true

Reply:
this is a nit pick, so it's not a blocker on my end. 
But I think 
`#[builder(default = "false")]` would be a better choice for this override, rather than using double-negative. 
Up to your taste

Reply:
I mean tests should be quick and dirty and easy to maintain, so I decided to keep it as little as possible. I would proceed with it as it is